feat(changelog): public-gateways-added-new-public-gateway-xl-offer 2024-12-03 (#4070)
scaleway/docs-content: changelog/december2024/2024-12-03-public-gateways-added-new-public-gateway-xl-offer.mdx

---
title: New Public Gateway XL offer
status: added
author:
    fullname: 'Join the #public-gateway channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-12-03
category: network
product: public-gateways
---

> We have just released the Public Gateway XL, a new offer with up to 10 Gbps of bandwidth for your most network-intensive workloads. Egress is included. Available in PAR-1 and PAR-2 only at the moment.

Reviewer: ```suggestion
We have now released the Public Gateway XL, a new offer with up to 10 Gbps of bandwidth for your most network-intensive workloads. Egress is included. Currently, this offer is available in PAR-1 and PAR-2 only.
```
